### DISCUSSION TITLE
Welcome to tiktok-src Discussions!

### DISCUSSION THREAD

**Discussion by wahdalo** [wahdalo/tiktok-src#16] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by arraya20** [comment]
Sore bang, mungkin bisa ditambahin fitur auto flip 👍🏻

**Comment by halowahyudi** [comment]
auto repost to our tiktok account
